release-notes/1.0/RC2_Contributor_List.md

```diff
@@ -0,0 +1,404 @@
+* [Austin Wise (1)](https://github.com/dotnet/coreclr/commits/master?author=AustinWise@gmail.com)
+* [Andy Ayers (1)](https://github.com/dotnet/coreclr/commits/master?author=andya@micrsoft.com)
+* [AlexGhiondea (1)](https://github.com/dotnet/coreclr/commits/master?author=AlexGhiondea@users.noreply.github.com)
+* [Maks Naumov (1)](https://github.com/dotnet/coreclr/commits/master?author=maksqwe1@ukr.net)
+* [Andrew Schwartzmeyer (1)](https://github.com/dotnet/coreclr/commits/master?author=andschwa@microsoft.com)
+* [Tijoy Tom Kalathiparambil (1)](https://github.com/dotnet/coreclr/commits/master?author=tijoytk@microsoft.com)
+* [taylorjonl (1)](https://github.com/dotnet/coreclr/commits/master?author=taylorjonl@gmail.com)
+* [Vance Morrison (1)](https://github.com/dotnet/coreclr/commits/master?author=vancemorrison@comcast.net)
+* [Josh Varty (1)](https://github.com/dotnet/coreclr/commits/master?author=JoshVarty@users.noreply.github.com)
+* [Jostein Kjønigsen (1)](https://github.com/dotnet/coreclr/commits/master?author=jostein@kjonigsen.net)
+* [Andrew Karpov (1)](https://github.com/dotnet/coreclr/commits/master?author=andrewcarpov@gmail.com)
+* [Andreas Strid (1)](https://github.com/dotnet/coreclr/commits/master?author=andreas_strid@bredband.net)
+* [Yi Zhang (CLR) (1)](https://github.com/dotnet/coreclr/commits/master?author=yzha@microsoft.com)
+* [Zlatko Knezevic (1)](https://github.com/dotnet/coreclr/commits/master?author=zlakne@microsoft.com)
+* [Eric Eilebrecht (1)](https://github.com/dotnet/coreclr/commits/master?author=ericeil@users.noreply.github.com)
+* [bartonjs (1)](https://github.com/dotnet/coreclr/commits/master?author=jbarton@microsoft.com)
+* [Drew Scoggins (1)](https://github.com/dotnet/coreclr/commits/master?author=anscoggi@microsoft.com)
+* [Rama (1)](https://github.com/dotnet/coreclr/commits/master?author=ramarag@microsoft.com)
+* [DionYe (1)](https://github.com/dotnet/coreclr/commits/master?author=junye@microsoft.com)
+* [Eugene Zemtsov (1)](https://github.com/dotnet/coreclr/commits/master?author=e.zemtsov@gmail.com)
+* [Ludovic Henry (1)](https://github.com/dotnet/coreclr/commits/master?author=ludovic@xamarin.com)
+* [Felipe Pessoto (1)](https://github.com/dotnet/coreclr/commits/master?author=felipepessoto@hotmail.com)
+* [Lakshmi Priya (1)](https://github.com/dotnet/coreclr/commits/master?author=Priya91@users.noreply.github.com)
+* [noahfalk (1)](https://github.com/dotnet/coreclr/commits/master?author=noahfalk@users.noreply.github.com)
+* [Ron Cain (1)](https://github.com/dotnet/coreclr/commits/master?author=roncain@microsoft.com)
+* [Chris (1)](https://github.com/dotnet/coreclr/commits/master?author=Chrisboh@microsoft.com)
+
+## CoreFX
+
+* [Stephen Toub (798)](https://github.com/dotnet/corefx/commits/master?author=stoub@microsoft.com)
+* [stephentoub (356)](https://github.com/dotnet/corefx/commits/master?author=stoub@microsoft.com)
```

Comment:
@leemgs, I believe the list includes commits made to master through March 21st, which is why your entry is in the list below with (4).